tool/gravity/cli/install.go

```diff
@@ -743,6 +746,29 @@ func completeJoinPlan(localEnv, joinEnv *localenv.LocalEnvironment, operation *o
 	return joinFSM.Complete(trace.Errorf("completed manually"))
 }
 
+func completeJoinPlanFromAnotherNode(localEnv *localenv.LocalEnvironment, operation *ops.SiteOperation) error {
```

Comment:
I'd maybe call it "completeJoinPlanFrom*Existing*Node" and add a comment explaining what this is for.